src/NServiceBus.Core.Tests/API/approvals/NServiceBus.Core.approved.cs

```diff
@@ -2244,7 +2239,6 @@ namespace NServiceBus.Unicast.Routing
     public class StaticMessageRouter
     {
         public StaticMessageRouter(System.Collections.Generic.IEnumerable<System.Type> knownMessages) { }
-        public bool SubscribeToPlainMessages { get; set; }
```

Comment:
Side note: I've pushed a change to apicomparer that lets you compare your "c./dev/NServiceBus/Binaries/NServiceBus.Core.dll" against the latest on the build server.

That call creates a nice report on whats been changed that at least for me is invaluable to detect and put the needed obsoletes in

https://github.com/ParticularLabs/APIComparer/commit/e99bcac6da34755a181408fbdeac18bf8b84b061